atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -131,6 +131,27 @@ def get_smtp_session(workflow, fallback):
     return smtplib.SMTP(config['host'])
 
 
+def get_cachito(workflow):
+    return get_value(workflow, 'cachito', NO_FALLBACK)
+
+
+def get_cachito_session(workflow):
+    config = get_cachito(workflow)
+    from atomic_reactor.cachito_util import CachitoAPI
+
+    api_kwargs = {'insecure': config.get('insecure', False)}
+
+    ssl_certs_dir = config['auth'].get('ssl_certs_dir')
+    if ssl_certs_dir:
+        cert_path = os.path.join(ssl_certs_dir, 'cert')
+        if os.path.exists(cert_path):
+            api_kwargs['cert'] = cert_path
+        else:
+            raise KeyError("Cachito ssl_certs_dir doesn't exist")
```

Comment:
IMHO it should be `ValueError`

Reply:
Or `RuntimeError`

Reply:
It's `KeyError` because that's what `get_value` raises if a config for `cachito` doesn't exist at all. I suppose `get_odcs_session` does this so it's easier to use by only catching a single exception.

Reply:
You're right though. `KeyError` doesn't make any sense. I changed it to `RuntimeError`.